### DISCUSSION TITLE
would like to invite new users to register and login into Weblate only using their GITHUB

### DISCUSSION THREAD

**Discussion by spurnasai** [WeblateOrg/weblate#11849] [General]
Is there is any env so that I can use and only allow the users to login via GITHUB and disable the Register new account completely                   I have used                                                                                                                                              
       WEBLATE_REGISTRATION_OPEN: 0 and 1
      WEBLATE_REGISTRATION_ALLOW_BACKENDS: github-oauth2  as env variables
and I have used this in my settings.py                                                                                                                                                      
 # Authentication configuration
AUTHENTICATION_BACKENDS: tuple[str, ...] = (
    "social_core.backends.github.GithubOAuth2",
)
# GitHub auth
SOCIAL_AUTH_GITHUB_KEY = get_env_str("WEBLATE_SOCIAL_AUTH_GITHUB_KEY")
if SOCIAL_AUTH_GITHUB_KEY:
    SOCIAL_AUTH_GITHUB_SCOPE = ["user:email"]
    SOCIAL_AUTH_GITHUB_SECRET = get_env_str(
        "WEBLATE_SOCIAL_AUTH_GITHUB_SECRET", required=True
    )
If I am using those env at that time the new users are unable to login it is saying that new registrations are not permitted If I am removing from the env then the users are able to login

**Comment by nijel** [comment]
Duplicate of https://github.com/WeblateOrg/weblate/issues/11850.
